source/JointMilitarySymbologyLibraryCS/jmsml.config

```diff
@@ -161,5 +161,409 @@
       <AmplifierValue ID="REDUCED" Label="-">1</AmplifierValue>
       <AmplifierValue ID="REINFORCED_REDUCED" Label="+/-">2</AmplifierValue>
     </AmplifierValues>
+
+    <SchemaTypes>
```

Comment:
General note/observation: this custom config file is becoming somewhat large and unwieldy and beyond the type of application-configuration information one might normally expect.

I know you mentioned that you put this database information (& the point ordering rules above) here because it is implementation-specific but creating a database of symbolset attributes dictated by the standard is something multiple users might want to do, even if the underlying RDBMS differs.